src/findings/finding_container.py

```diff
@@ -49,6 +49,27 @@ def getAllFindings(cls):
     def getActionableFindings(cls):
         return [finding for finding in cls._finding_results if finding.actionable]
 
+    @classmethod
+    def toHumanReadableMessage(cls):
+        output_message = ""
+        file_to_findings = {}
+        for finding in cls.getActionableFindings():
+            # Create a map to summarize the findings based on proto file name.
+            if finding.location.proto_file_name not in file_to_findings:
+                file_to_findings[finding.location.proto_file_name] = []
+            file_to_findings[finding.location.proto_file_name].append(finding)
```

Comment:
You can use a `defaultdict` and save a few lines.
```python
file_to_findings = defaultdict(list)
for finding in cls.getActionableFindings():
    file_to_findings[finding.location.proto_file_name].append(finding)
```

Reply:
Sure! Just learned the useful usage for defaultdict from another PR, thanks!

Reply:
They're my favorite data structure :)